wlauto/workloads/googlephotos/__init__.py

```diff
@@ -0,0 +1,90 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from wlauto import AndroidUxPerfWorkload, Parameter, File
+from wlauto.exceptions import ValidationError
+from wlauto.utils.types import list_of_strings
+from wlauto.utils.misc import unique
+
+
+class Googlephotos(AndroidUxPerfWorkload):
+
+    name = 'googlephotos'
+    package = 'com.google.android.apps.photos'
+    min_apk_version = '1.21.0.123444480'
+    max_apk_version = '1.26.0.130713109'
```

Comment:
That was the latest version when I committed. I see that a new version came out on the 2nd of September.